client/consul/sync.go

```diff
@@ -133,15 +132,26 @@ func (c *ConsulService) SetDelegatedChecks(delegateChecks map[string]struct{}, c
 	return c
 }
 
-// SyncTask sync the services and task with consul
-func (c *ConsulService) SyncTask(task *structs.Task) error {
+// SetAddrFinder sets a function to find the host and port for a Service
```

Comment:
// SetAddrFinder sets a function to find the host and port for a Service given its service identifier